Olympus.Stratis/functions/fn_resetDamagedObjects.sqf

```diff
@@ -29,15 +29,16 @@ private _action = [
                 private _type = typeOf _x;
                 deleteVehicle _x;
 
-                // Run this in the next frame
+                // Waits 2s then executes.
```

Comment:
```suggestion
                // Run after 2 seconds
```
Obviously it executes, that's a redundant comment.

Why was this changed to 2 seconds though?

Reply:
There were still vehicles flying off into outer space, removed the comment.